R/expectations_xl.R

```diff
@@ -20,6 +20,8 @@
 #' @param format \code{[character]} Comparison mode passed to \code{diffPrint},
 #' defaults to to using the \dQuote{diffobj.format} global option value with
 #' \dQuote{ansi256} as fallback if no such option is set
+#' @param strict \code{[logical]} scalar. If set to \code{TRUE}, any exception 
+#'        worse than the wanted exception will cause the test to fail.
```

Comment:
Methinks you forget to a) add it the signature of the function(s) and b) possibly the calls and c) the generated manual page (I can do the lattter if roxygen shows side effects of rewriting).

Reply:
It also lacks a default value.  Should it be `TRUE` in the call?  Or another value?  Please advise.

Reply:
The default value is `FALSE` in `tinytest` (for back-compatibility). I was under the (possibly false) impression that the signature and default values are automatically imported via the call to `supersize`

Reply:
I think I should have been more explicit. The `strict` argument was only added to `expect_warning` and `expect_message`. So no need to add them to `expect_equivalent` etc. As these are imported through `supersize` I _think_ this is all automatic. I'll do a local change and test against the development version of `tinytest`

Reply:
Yes, I corrected the PR for this hours ago [in this commit](https://github.com/eddelbuettel/ttdo/pull/8/commits/5c193ce725c24bffe45f6e52ce6683a5a36008c7See) 

I also realized later that we probably could have added this simply as a simply argument `strict` without a value follow by `if (missing(strict)) strict <- FAlSE` which might have been smoother upgrade path.  Too late now.